pkg/artifactory/resource_artifactory_single_replication_config.go

```diff
@@ -3,10 +3,12 @@ package artifactory
 import (
 	"context"
 	"fmt"
+	"net/http"
+
 	"github.com/atlassian/go-artifactory/v2/artifactory"
 	"github.com/atlassian/go-artifactory/v2/artifactory/v1"
+	v1 "github.com/atlassian/go-artifactory/v2/artifactory/v1"
```

Comment:
It looks like your tests in Travis are failing due to the inclusion of this line. It's the same as the previous line. If you try removing it, the test should run again

For reference the error in Travis was
```
pkg/artifactory/resource_artifactory_single_replication_config.go:10:2: v1 redeclared as imported package name
	previous declaration at pkg/artifactory/resource_artifactory_single_replication_config.go:9:2
FAIL	github.com/atlassian/terraform-provider-artifactory/pkg/artifactory [build failed]
```

Reply:
I've removed the duplicate import. My editor really wants to be "helpful" and add the alias for v1 even though it doesn't really matter. I can remove it if you want.